### DISCUSSION TITLE
Card Set

### DISCUSSION THREAD

**Discussion by hchan-acara** [Carduelis/poker-widget-preact#10] [Q&A] [answered]
Is there a way to change the card set e.g. remove 0 and 1/2 and change 20 to 21?

**Comment by AdrianTheel** [comment]
Is there a possibility to estimate also about other sets, e.g. about T-shirt sizes?

**Reply by Carduelis** [reply, marked as answer]
Hello! 
It's always better late than never, but we are working on the new features to add to the application. 
T-shirts and fibonnachi are on the way
